packages/core/contracts/cross-chain-oracle/chain-adapters/Optimism_ParentMessenger.sol

```diff
@@ -58,7 +58,7 @@ contract Optimism_ParentMessenger is CrossDomainEnabled, ParentMessengerInterfac
     function setChildParentMessenger(address newParentMessenger) public onlyOwner {
         bytes memory dataSentToChild = abi.encodeWithSignature("setParentMessenger(address)", newParentMessenger);
         sendCrossDomainMessage(childMessenger, defaultGasLimit, dataSentToChild);
-        emit MessageSentToChild(dataSentToChild, address(0), defaultGasLimit, childMessenger);
+        emit MessageSentToChild(dataSentToChild, childMessenger, defaultGasLimit, childMessenger);
```

Comment:
I intentionally set this to the `address(0)` btw. reason is this prop is the `targetSpoke`. in this case, there is no `targetSpoke` as this is changing the `childMessenger`. does it not make more sence to be blank (i.e 0x0) to indicate there is no _target spoke_.

It might help to rename the 4th param in `MessageSentToChild` from `childAddress` to `childMessengerAddress`.

Reply:
hmm but isn't this more of a `targetContract`, where the final execution happens? Perhaps thats meaningless

Reply:
I'll revert and rename to `childMessenger`